src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingController.kt

```diff
@@ -47,6 +47,12 @@ class EksternBehandlingController(
         return Ressurs.success(eksternBehandlingService.harLøpendeStønad(personidenter))
     }
 
+    @PostMapping("har-loepende-barnetilsyn")
+    @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
+    fun harLøpendeBarnetilsyn(@RequestBody personIdent: String): Ressurs<Boolean> {
```

Comment:
Antar dette er ifm sjekk om noen har barnetilsyn når det fattes et vedtak om kontantstøtte?
Litt usikker, men er "løpende" dekkende her? Eller burde det sendes inn periode fra og periode til?

Et kontantstøttevedtak kan jo være frem/bak i tid, og ikke gjelde "nå". Følgelig bør vi jo sjekke barnetilsyn for den perioden man får innvilget kontantstøtte, kanskje?

Reply:
Godt poeng 🤔 Ja, kanskje vi bør sjekke om perioden for kontantstøtte overlapper med barnetilsyn, og ikke bare sjekke om man har barnetilsyn "nå" / løpende.

Reply:
Prata med Lars, og vi ble enige om å bare gå for "nåtid", og så får vi se om vi skal sjekke overlapp av perioder senere.